### DISCUSSION TITLE
Boxes is hanging quiet often

### DISCUSSION THREAD

**Discussion by elSolus** [elementary/triage#112] [Q&A] [answered]
Hi. I installed GNOME Boxes from the AppCenter. First, at the time of installing it stuck on, like, 99% for a long time. Then, every now and then when I open the app, sometimes it stuck on "adding new" OS or often when I minimize the app with OS(es) still running (a notification says xyz in running in the bg), it doesn't open again, even when you go to the Applications menu and click on Boxes there. It just stay away ("minimized" I guess)

**Comment by danirabbit** [comment, marked as answer]
elementary doesn't maintain the Boxes app. It's maintained by GNOME, so you'll need to report any issues with it to GNOME :)
